src/services/composite-keys-manager.js

```diff
@@ -1,54 +1,92 @@
 import _ from 'lodash';
+import { Schemas } from 'forest-express';
 import Operators from '../utils/operators';
 
-function CompositeKeysManager(model, schema, record) {
-  const GLUE = '|';
+/**
+ * This helper class allows abstracting away the complexity
+ * of using collection which have composite primary keys.
+ */
+class CompositeKeysManager {
+  constructor(model, glue = '|') {
```

Comment:
It makes absolutely no sense to use another glue parameter (and would probably break things).
It's a hack so that Eslint stop bugging me about the rule "you must use `this` in all methods, otherwise it should be static"

Reply:
Sorry but i can't let this one pass 💇